index.html

```diff
@@ -6029,8 +6029,13 @@ <h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
           Otherwise, if a user agent provides an implicit <sref>aria-selected</sref> value for an <rref>option</rref>, the value SHOULD be <code>false</code>.
         </p>
         <p>
-          Authors MAY indicate selection for <rref>option</rref> elements  using either <sref>aria-selected</sref> or <sref>aria-checked</sref>.
-          Some user interfaces indicate selection with <sref>aria-selected</sref> in single-select list boxes and with <sref>aria-checked</sref> in multi-select list boxes.
+          Authors SHOULD indicate selection for <rref>option</rref> elements using one of the following:
+        </p>
+        <ul>
+          <li>An <sref>aria-selected</sref> value of <code>true</code> on the selected option within a single-select <rref>listbox</rref>, and optionally <sref>aria-selected</sref> values of <code>false</code> on unselected options.</li>
+          <li>Either <sref>aria-selected</sref> or <sref>aria-checked</sref> on all options within a multi-select <rref>listbox</rref></li>
```

Comment:
Suggest being explicit with the `true` or `false` values to be inline with the previous list item indicating that authors should use a value of true.  Additionally, if not being specific here there's the potential for someone to assume that `mixed` or `undefined` are allowed... 

```suggestion
          <li>Either <sref>aria-selected</sref> or <sref>aria-checked</sref> on all options within a multi-select <rref>listbox</rref>, with the attributes' specified values of `true` or `false`, representing their current selected or unselected state.</li>
```

Reply:
I wordsmithed a bit, let me know if you like the current wording